src/components/Profile/ProfileContainer.jsx

```diff
@@ -8,14 +8,14 @@ import {getUserProfile, getStatus, updateStatus} from '../../Redux/profile-reduc
 import Profile from './Profile';
 
 const ProfileContainer = props => {
+
   useEffect(() => {
-    let userId = props.match.params.userId;
+    const userId = props.match.params.userId ? props.match.params.userId : props.authorizedUserId;
```

Comment:
now I can see that it can be simpler
`props.match.params.userId || props.authorizedUserId;`